CHANGELOG.md

```diff
@@ -11,10 +11,11 @@
 * Renaming symbol fails within a file that had recently been renamed without saving changes.
   * As a workaround, make an edit within the file before using Rename Symbol.
 
-## 1.23.3 (Not yet released)
+## 1.23.4 (Not yet released)
```

Comment:
Why 4?

Reply:
I thought releases once created cannot be modified.

Reply:
I think we update the pre-release beta version, without any changes here.
https://github.com/OmniSharp/omnisharp-vscode/releases/tag/v1.23.3-beta1